### ISSUE TITLE
Disable confirmation of kill-session in choose-tree when pressing X.

### ISSUE THREAD

**Issue by f0s3** [tmux/tmux#3823]
### Issue description

Basically, the title.

If it's possible to bind what I want to a shortcut - let me know. I tried binding custom action to Leader-x that will kill currently selected item in choose-tree, but I cannot seem to find a way to get selected items' session name. I looked into what variables are available, haven't found anything useful.

### Required information
* tmux 3.3a
* EndeavourOS, arch derivative
other info is unrelated to the issue

**Comment by krader1961** [comment]
FWIW, Your subject line and issue description are different, AFAICT. Your description implies you can't find a way to bind an action that will kill a session selected from the choose-tree list. But your subject line implies you are able to do so but want to avoid confirming the session kill.

You should also show us the binding(s) you have tried.

**Comment by f0s3** [comment]
@krader1961 I want to achieve the behaviour I described: no additional dialog when closing the session. It doesn't matter how: through a shortcut, or being a new flag that I can disable in the config file. It doesn't matter which shortcuts I have tried, since there's nothing that worked.

But basically I thought of something like this:

```
bind -n M-x kill-session -t $selected_session_name
```
where `$selected_session_name` is the name of the session, currently selected in choose-tree.

Basically I want to somehow close selected session without executing [window-tree.c:1290](https://github.com/tmux/tmux/blob/ea7136fb838a2831d38e11ca94094cea61a01e3a/window-tree.c#L1290)

If you or anyone happen to know a way to achieve this - let me know.
